dangerzone/logic.py

```diff
@@ -69,8 +69,10 @@ def convert_doc(document: Document) -> None:
             )
 
         max_jobs = self.isolation_provider.get_max_parallel_conversions()
-        with concurrent.futures.ThreadPoolExecutor(max_workers=max_jobs) as executor:
-            executor.map(convert_doc, self.documents)
+        # with concurrent.futures.ThreadPoolExecutor(max_workers=max_jobs) as executor:
+        #    executor.map(convert_doc, self.documents)
+        for document in self.documents:
+            convert_doc(document)
```

Comment:
There was nothing wrong with the original approach:
```suggestion
        with concurrent.futures.ThreadPoolExecutor(max_workers=max_jobs) as executor:
           executor.map(convert_doc, self.documents)
```